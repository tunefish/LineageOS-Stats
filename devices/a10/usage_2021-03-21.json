{
  "16.0": 53,
  "17.1": 776,
  "18.1": 250,
  "none": 280
}