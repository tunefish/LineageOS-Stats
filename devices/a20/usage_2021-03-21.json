{
  "17.1": 2426,
  "18.1": 136,
  "none": 24
}