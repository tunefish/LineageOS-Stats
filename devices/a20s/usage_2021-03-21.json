{
  "17.1": 2485,
  "none": 1
}