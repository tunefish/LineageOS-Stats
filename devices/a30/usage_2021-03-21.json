{
  "17.1": 32,
  "18.0": 712,
  "18.1": 336,
  "none": 4
}