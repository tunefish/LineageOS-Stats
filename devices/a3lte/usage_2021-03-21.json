{
  "15.1": 674,
  "16.0": 243,
  "17.1": 778
}