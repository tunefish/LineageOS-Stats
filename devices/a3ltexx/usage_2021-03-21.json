{
  "14.1": 1052
}