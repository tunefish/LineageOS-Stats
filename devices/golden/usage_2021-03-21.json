{
  "13.0": 1185,
  "14.1": 4020,
  "15.1": 5,
  "17.1": 8,
  "none": 1
}